### ISSUE TITLE
Files with an m4a extension are incorrectly identified as "video/quicktime"

### ISSUE THREAD

**Issue by brenogazzola** [rails/marcel#57]
Take a `m4a` file and ask Marcel for its content_type:
```ruby
irb(main):007:0> Marcel::MimeType.for(Pathname.new "/Users/brenogazzola/Desktop/voice.m4a")
=> "video/quicktime"
```

Which is strange since the `tables.rb` file has the correct mime type: `'m4a' => 'audio/mp4',`

**Comment by brenogazzola** [comment]
I checked the code and saw there are tests specifically for identifying M4a files using magic bytes and they are passing. So I used `ffprobe` to get a bit more info about the file and found this:
```
Metadata:
    major_brand     : 3gp4
    minor_version   : 0
    compatible_brands: isom3gp4
    creation_time   : 2021-06-07T18:59:23.000000Z
    com.android.version: 10
```

It seems this file was recorded using Android's voice recorder. So I tested and got a file that fails the test if you place it in the folder `fixtures/magic/audio/mp4`. Here's a [dropbox link](https://www.dropbox.com/s/anrqqbpgbdalqlx/android_voice_recorder.m4a?dl=0) for the file.

I tried to figure out what was the cause of the problem, but this whole "magic bytes" thing is a bit too magic for me...
